lib/config/definitions.js

```diff
@@ -1150,6 +1150,26 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'java',
+    description: 'Configuration object for all Java package managers',
```

Comment:
Right now this is gradle-only

Reply:
The configuration I've added is the one below for gradle. Not sure if we need this one as well?

Reply:
It's standard convention in Renovate to include a `language` even if there is only a single package manager implemented so far. Later, this `java` object could be used to share common config for Maven and Gradle, for instance.